nrf_rpc/Kconfig

```diff
@@ -26,6 +26,13 @@ config NRF_RPC_TR_RPMSG
 	help
 	  If enabled selects RPMsg as a transport layer for nRF PRC.
 
+config NRF_RPC_TR_SHMEM
+	bool "nRF RPC over shared memory"
+	select SM_IPT
+	help
+	  If enabled selects shared memory as a transport layer for nRF PRC.
+	  It requires additional API implemented in nrf_rpc_os.h file.
+
```

Comment:
We are using SM IPT module, so we can use its name instead of just `shared memory`.